src/controller/osw-controller.ts

```diff
@@ -308,7 +308,8 @@ class OSWController implements IController {
             const params = new FeedbackDownloadRequestParams(JSON.parse(JSON.stringify(initParams)));
             await params.validateRequestInput();
             const stream = await oswService.downloadFeedbacks(params);
-            response.setHeader('Content-Type', 'text/csv');
+            const contentType = params.format === 'geojson' ? 'application/geo+json' : 'text/csv';
```

Comment:
Please check the condition in unit testing where user may pass format as `CSV` or `GeoJSON` instead of `csv` or `geojson`